app/Http/ViewComposers/MenuComposer.php

```diff
@@ -62,7 +62,7 @@ public function compose(View $view)
         // end global Settings
 
         //TODO: should be handled via CSS Themes
-        $vars['navbar'] = in_array($site_style, ['mono', 'dark']) ? 'navbar-inverse' : '';
```

Comment:
if this is no longer needed it should be removed from all templates and so that use it.

Reply:
agree, i dont know where its used that variable... so i didnt removed it completely.

Reply:
Remove it and search for {{ $navbar}} to start with.

Reply:
if it's really not more used, for my intention TODO-Line should be removed also

Reply:
Already fixed it, its needed by the "mono"theme... so i left that sentence, with "mono" only.